Packs/ApiModules/ReleaseNotes/2_2_31.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### CoreXQLApiModule
+
+Improve the way of handling the scenario when the maximum allowed number of parallel running queries is reached by using a polling mechanism instead of throwing an error.
```

Comment:
```suggestion
Improved the handling of scenarios where the maximum allowed number of parallel running queries is reached by using a polling mechanism instead of throwing an error.
```